source/includes/write/delete.kt

```diff
@@ -0,0 +1,35 @@
+import com.mongodb.client.model.DeleteOptions
+import com.mongodb.client.model.Filters.*
+import com.mongodb.kotlin.client.MongoClient
+
+// start-data-class
+data class Restaurant(val name: String, val borough: String)
+// end-data-class
+
+fun main() {
+    val uri = "<connection string>"
+
+    val mongoClient = MongoClient.create(uri)
+    val database = mongoClient.getDatabase("sample_restaurants")
+    val collection = database.getCollection<Movie>("restaurants")
+
+    // start-delete-one
+    val filter = eq(Restaurant::name.name, "Happy Garden")
+    val result = collection.deleteOne(filter)
+    // end-delete-one
+
+    // start-delete-many
+    val filter = and(
+        eq(Restaurant::name.name, "Happy Garden"),
```

Comment:
Based on the introduction you give to this example in `delete.txt`:

```suggestion
        eq(Restaurant::borough.name, "Brooklyn"),
```